src/main/java/nextstep/subway/section/domain/Distance.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.subway.section.domain;
+
+import javax.persistence.Embeddable;
+
+@Embeddable
+public class Distance {
+
+    int value;
```

Comment:
외부에서 데이터에 접근할 수 있지 않을까요?

```suggestion
    private int value;
```